cypress/e2e/dashboard.cy.ts

```diff
@@ -0,0 +1,205 @@
+describe("Dashboard", () => {
+  type KeyType = {
+    [key: string]: string;
+  };
+
+  const keys: KeyType = {
+    unfundedPublicKey: Cypress.env("UNFUNDED_PUBLIC_KEY") || "",
+    unfundedSecretKey: Cypress.env("UNFUNDED_SECRET_KEY") || "",
+    fundedPublicKey: Cypress.env("FUNDED_PUBLIC_KEY") || "",
+    fundedSecretKey: Cypress.env("FUNDED_SECRET_KEY") || "",
```

Comment:
We should be more explicit about this. What account is this? Source Account? Destination?

Reply:
I have renamed these variables to make them more descriptive.